src/jquery.data-remote.js

```diff
@@ -177,7 +196,7 @@
         var method = _options.oneAndDone ? 'one' : 'on';
 
         // bind to specific event type
-        $element[method](_options.eventType, function(evt) {
+        $element[method](_options.eventType, debounce(function(evt) {
```

Comment:
which event types would we not want a debounce on? for instance, the 'load' event type is covered on line 185 so I assumed what's left are mostly the keys/scrolling etc, things that are physically interacted with.